### DISCUSSION TITLE
Android 10

### DISCUSSION THREAD

**Discussion by Truenoxdx1** [skyline-emu/skyline#1267] [General]
En android 10 hay muchas fallas les podria pedir de favor que las arreglarán por favor como el cuphead se pone pantalla blanca y no me deja jugar y tengo un snapdragon 855 y 6 de ram y no me deja se queda en blanco la pantalla les pediría de favor que lo arreglarán

**Comment by Joraganlas** [comment]
I think for Android 10 there is still a problem with the graphics driver, I also experienced the same problem when testing using Android 10, but if using Android 11 it's fine, maybe in the future developers can develop this emulator so that it can also be used on Android 10 so that  no problem with the graphic bug again, thank you

**Reply by Truenoxdx1** [reply]
Ah ya ok gracias

**Comment by w4w2w4** [comment]
android 10  bug
